### PR TITLE
Fix cassandra 30 row cache

### PR DESCRIPTION
**Pull request by martincsimpson**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [X ] Update Changelog following the conventions laid out in [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [x] Update README with any necessary configuration snippets

- [x] Binstubs are created if needed

- [X ] RuboCop passes

- [ ] Existing tests pass

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins
 
 ## [Unreleased]
 
+## [2.1.1] - 2019-03-04
```
Comment (nixwiz) [inline CHANGELOG.md]: Can you please move your changelog entry to Unreleased.  We will move it to the released version when a new release is created.

File: lib/sensu-plugins-cassandra/version.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@ module SensuPluginsCassandra
   module Version
     MAJOR = 2
     MINOR = 1
-    PATCH = 0
```
Comment (nixwiz) [inline lib/sensu-plugins-cassandra/version.rb]: Can you please remove this change, again we will update version numbers when we roll up a release.

Comment (martincsimpson) [reply lib/sensu-plugins-cassandra/version.rb]: This has been done

File: CHANGELOG.md (hunk 1 above)
Comment (martincsimpson) [reply CHANGELOG.md]: Also done 👍

**Review by nixwiz — APPROVED** [review]

**Pull request merged**